### ISSUE TITLE
Adding point to route that route is already using is not possible. 

### ISSUE THREAD

**Issue by koebi** [GIScience/ors-map-client#88]
Currently, the client doesn't support adding a point to a route via right-click if it is already routing over that point.

Especially when creating a particular route using waypoints, this behaviour is quite unexpected and I think it should be supported :)

**Comment by amoncaldas** [comment]
Hi @koebi 
Could you please indicate the steps to reproduce this?
Thanks

**Comment by koebi** [comment]
Hey,
yes of course :)

I went for a walk with a friend and I wanted to map out the route we took to find out how far we had walked.
I found the area on the client and started adding waypoints to our route:
![maps_client_issue88](https://user-images.githubusercontent.com/4692974/108980756-50eadb80-768c-11eb-9097-e695c42bc790.png)

All of these waypoints were added by right-clicking the map in the chosen point and then selecting the appropriate option from the right-click menu (1 -> Directions from here, 2 -> Directions to here, 3+4 -> append as destination).
The white dot on the rightmost part of the route represents the next point that I'd like to add to my route, but since the route is already using that point, I can't right-click on it.
